test/services/xccdf_report_parser_test.rb

```diff
@@ -112,6 +112,7 @@ class XCCDFReportParserTest < ActiveSupport::TestCase
 
   context 'rule results' do
     should 'save them, associate them with a rule and a host' do
+      @report_parser.stubs(:save_rule_references)
```

Comment:
This significantly speeds up this test (Jenkins was sometimes killing it due to long runtime ~150s), and `save_rule_references` is not relevant for this test.